docs/source/configuration/configuration-partition.rst

```diff
@@ -27,7 +27,7 @@ For users who are already familiar with DGL and know how to construct DGL graph,
 .. Note::
     The sum of the **train-pct** and **val-pct** should be less than 1. And the percentage of test nodes/edges is the result of 1-(train_pct + val_pct).
 
-- **--undirected**: if add this argument, will add reverse edges to the given graph.
+- **--add-reverse-edges**: if add this argument, will add reverse edges to the given graph.
```

Comment:
Could you also change the corresponding strings at wiki page https://github.com/awslabs/graphstorm/wiki/configuration-configuration-partition?